### PR TITLE
Initial Angular template

### PR DESCRIPTION
**Pull request by mkinsey**
This PR adds resources used throughout the project (bootstrap, jQuery, AngularJS) , establishes a directory structure, and includes a base Angular template that can be revised or expanded upon.

### CODE REVIEW

File: public/css/bootstrap-theme.css [hunk 1]
```diff
@@ -0,0 +1,587 @@
+/*!
```
Comment (benjic) [inline public/css/bootstrap-theme.css]: A minified version or CDN link may be easier then committing this all to source.

**Comment by benjic** [comment]
Have you put any thought into using a dependency management tool for bootstrap and angular? Like Bower or NPM? This would simplify getting these kinds of assets and would allow for version management across repos.

**Comment by benjic** [comment]
@mkinsey You can install the Gulp and Bower programs and  with:
```
[sudo] npm install -g gulp bower
npm install && bower install
```

Once dependencies are installed you can launch a test webserver which hosts an example of the angular application.
```
glup
```

I used the following resources regarding modularizing angular applications:

* http://henriquat.re/modularizing-angularjs/modularizing-angular-applications/modularizing-angular-applications.html
* http://briantford.com/blog/angular-bower

**Comment by mkinsey** [comment]
@orbitable/frontend Let me know if you have trouble getting started etc.

**Comment by benjic** [comment]
:+1:

**Pull request merged**